praktikumsplaner-frontend/frontend/src/components/common/StudienrichtungOrAusbildungsrichtungSelect.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <v-container>
+        <v-row>
+            <v-col>
+                <v-list-item>
+                    <v-select
+                        v-model="nwkProp.studiengang"
+                        label="Studienrichtung"
+                        :items="Studiengang"
+                        item-value="value"
+                        item-text="name"
+                        outlined
+                        clearable
+                        :rules="isStudiumOrAusbildungRule"
+                        @click:clear="nwkProp.studiengang = undefined"
+                    ></v-select>
+                </v-list-item>
+            </v-col>
+            <v-col>
+                <v-list-item>
+                    <v-select
+                        v-model="nwkProp.ausbildungsrichtung"
+                        label="Ausbildungsrichtung"
+                        :items="Ausbildungsrichtung"
+                        item-value="value"
+                        item-text="name"
+                        outlined
+                        clearable
+                        :rules="isStudiumOrAusbildungRule"
+                        @click:clear="nwkProp.ausbildungsrichtung = undefined"
```

Comment:
Even if its just one line of code, it should be put into a method so there is no/less logic in the template. Additionally if you need to do anything else when clearing for example showing a message or sending a request or trigger an event the code in the template would become hard to read and maintain.